### ISSUE TITLE
Memory?

### ISSUE THREAD

**Issue by Kataruu** [UnnamedOrange/osu-kps#12]
Hi,
I set the key monitor method to Memory but nothing is being registred, am I doing something wrong?

**Comment by UnnamedOrange** [comment]
Hello,
First, you should modify the keys as in the setting of osu.
Another possibility is that you are using an older version of osu-kps. osu's update on 2022-03-26 invalidated the ability to read memory properly. The [latest release](https://github.com/UnnamedOrange/osu-kps/releases/tag/beta-2023.328) has patched this issue.

**Comment by Kataruu** [comment]
Hi sorry for the late response,
I did exactly what you said above, I have exact same setting but it did not fix the problem.
And I do have the lastest release.

I also tried this:
- Switch from cuttingedge to stable
- Using older version of osu!
- Running the software in Administrator
- Run the software before/after osu! is launched
- Used a old version of osu-kps (beta-2022.816) and played with a version of osu! before the 20230326 update
